app/res/drawable/progress_bar.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It's not limited to audio recording - Could be used as a better looking alternative to the default Android progress bar anywhere.